CHANGES.md

```diff
@@ -4,10 +4,12 @@ Libplanet changelog
 Version 0.7.0
 -------------
 
-To be released.
+Released on October 1, 2019.
```

Comment:
Oh, could you left date this line as it was (i.e., <q>To be released.</q>)?  We should change this date when we actually make a new release (about a month later).

Reply:
Sure thing !

Reply:
Done !